dists/scummvm.rc

```diff
@@ -24,8 +24,8 @@ scummclassic.zip       FILE    "gui/themes/scummclassic.zip"
 scummmodern.zip        FILE    "gui/themes/scummmodern.zip"
 scummremastered.zip    FILE    "gui/themes/scummremastered.zip"
 residualvm.zip         FILE    "gui/themes/residualvm.zip"
-achievements.dat       FILE    "dists/engine-data/achievements.dat"
 encoding.dat           FILE    "dists/engine-data/encoding.dat"
+achievements.dat       FILE    "dists/engine-data/achievements.dat"
```

Comment:
It would be better to remove this change from the history, so it is not removed in the first place

Reply:
Removed